helpers/websocket.js

```diff
@@ -1,6 +1,6 @@
 /**
  * tags:
- * "users:request_status_update" = user request status updates
```

Comment:
Don't remove it, i made this to explain what a tag does

Reply:
tag is self-explanatory, no need to over-explain it